CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - ROX-23155: EBPF collection has been removed. If EBPF is configured, it
   will be automatically converted to CORE_BPF. forceCollection no longer has an
   effect.
+- The `ROX_CLOUD_SOURCES` feature flag has been removed. Its effect is now enabled by default.
```

Comment:
I don't think we've made this explicit in the past, and it has already been enabled for some time, so just remove it without adding changelog info?